test_runner/fixtures/neon_fixtures.py

```diff
@@ -1142,7 +1142,7 @@ def create_tenant(
         self,
         tenant_id: Optional[TenantId] = None,
         timeline_id: Optional[TimelineId] = None,
-        conf: Optional[Dict[str, str]] = None,
+        conf: Optional[Dict[str, Any]] = None,
```

Comment:
made compatible with `set_tenant_config` to re-use config object.

Reply:
I am unsure if I like this change, stringification has been with us, and it does help us places I think.

However now looking at the test case you fixed, I still haven't understood why.

Reply:
From the main thread:

> I fixed flacky test_issue_5878. I used a set_tenant_config with one option that reset the other options to defaults. Because of this, the compaction loop could be concurrently running with the manual compaction.

Ok I guess I am now starting to see it. This test also creates extra tenant for no reason.

Reply:
Is there a log of the flaky run?

Reply:
Apologies, I was quite confused because of the force-push and rewriting of the problematic commit. I'll try to fix this test case.